### ISSUE TITLE
[2.0.x] ResultSet#getInt returns 0 on a long

### ISSUE THREAD

**Issue by fbiville** [neo4j-contrib/neo4j-jdbc#57]
I'm not sure, but I think this violates the spec: http://download.oracle.com/otn-pub/jcp/jdbc-4_1-mrel-spec/jdbc4.1-fr-spec.pdf?AuthParam=1450886516_9f782c0a6b934f329a1d48bed1d51d87.

I think ResultSet#getInt on long should return a value at least when there is no overflow and maybe fail otherwise. WDYT?

**Comment by jexp** [comment]
Yes, you're right, it should use ((Number)value).intValue() I think.

wanna senda  PR ?

**Comment by fbiville** [comment]
Yep, will do tomorrow :)
